src/Writers/HTMLWriter.jl

```diff
@@ -891,13 +891,14 @@ end
 analytics_script(tracking_id::AbstractString) =
     isempty(tracking_id) ? Tag(Symbol("#RAW#"))("") : Tag(:script)(
         """
-        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-        })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-
-        ga('create', '$(tracking_id)', 'auto');
-        ga('send', 'pageview', {'page': location.pathname + location.search + location.hash});
+        <script async src="https://www.googletagmanager.com/gtag/js?id=$(tracking_id)"></script>
+        <script>
+          window.dataLayer = window.dataLayer || [];
+          function gtag(){dataLayer.push(arguments);}
+          gtag('js', new Date());
+
```

Comment:
```suggestion
          gtag('js', new Date());
          gtag('config', GA_MEASUREMENT_ID, {'page_path': location.pathname + location.search + location.hash});

```

Something like that? Based on https://developers.google.com/analytics/devguides/collection/gtagjs/single-page-applications

Reply:
Could you replace GA_MEASUREMENT_ID with $(tracking_id)?

Reply:
done!

Reply:
Thanks!